web-app/src/main/webapp/jasmine/spec/loadDataSpec.js

```diff
@@ -1,17 +1,64 @@
 import { filterReports } from "../../script/loadData.js";
 
-describe("Filtering timeframe ", () => {
-  const sampleData = [
+describe("Filtering ", () => {
+  const sampleDataOld = [
```

Comment:
please put the actual test code inside the `it` to make it clear what each test is actually testing. Note that your tests should work correctly regardless of the order they're run.